website/source/docs/agent/telemetry.html.md

```diff
@@ -92,6 +92,42 @@ These metrics are used to monitor the health of specific Consul agents.
     <td>number of objects</td>
     <td>gauge</td>
   </tr>
+  <tr>
+    <td>`consul.acl.cache_hit`</td>
+    <td>The number of ACL cache hits.</td>
+    <td>hits</td>
+    <td>counter</td>
+  </tr>
+  <tr>
+    <td>`consul.acl.cache_miss`</td>
+    <td>The number of ACL cache misses.</td>
+    <td>misses</td>
+    <td>counter</td>
+  </tr>
+  <tr>
+    <td>`consul.acl.replication_hit`</td>
+    <td>The number of ACL replication cache hits (when not running in the ACL datacenter).</td>
+    <td>hits</td>
+    <td>counter</td>
+  </tr>
+  <tr>
```

Comment:
These are already down in "cluster health", though they do make sense here. Maybe move that stuff up?